lambdas/export_rds_snapshot_to_s3/main.py

```diff
@@ -1,6 +1,5 @@
 import logging
```

Comment:
Looks behind line 7 needs adding 
`logger.setLevel(logging.INFO)`
otherwise, logger.info in the try and except block won't work, since the default logging level is "warning".  (see this [URL](https://docs.python.org/3/howto/logging.html#:~:text=The%20default%20level%20is%20WARNING%2C%20which%20means%20that%20only%20events%20of%20this%20level%20and%20above%20will%20be%20tracked%2C%20unless%20the%20logging%20package%20is%20configured%20to%20do%20otherwise.)).